content/customize-gnome.md

```diff
@@ -18,121 +18,138 @@ section: software
 tableOfContents: true
 ---
 
-**Please note**: Pop_OS! 22.04 is based on GNOME 42 so the extension **must** support GNOME 42.
 
-The GNOME desktop environment is highly customizable.  With a few extensions, the desktop can look and feel like many other operating system's desktops.  Launcher bars can be added, window lists can be used, and the overall look and feel of the desktop can be changed.
+The GNOME desktop environment is highly customizable. Visit [extensions.gnome.org](https://extensions.gnome.org) to install, remove, and configure currently installed extensions.  A web browser plugin is needed to interact with the site, and it will prompt for installation if needed.  Remember to refresh the page after the plugin is installed.
 
-To install a customization, simply head to [extensions.gnome.org](https://extensions.gnome.org) to install, remove, and configure currently installed customizations.  A web browser plugin is needed to interact with the site, and it will prompt for installation if needed.  Remember to refresh the page after the plugin is installed.
+**Please note**: Pop_OS! 22.04 is based on GNOME 42 so the extension **must** support GNOME 42. There are many extensions to choose from! If one isn't currently working, or breaks with an update, then try a different extension to see if it works better.
 
-We will share some of the more popular extensions, but there are many more to choose from!  If one is found and it isn't currently working, or breaks with an update, then try a different extension to see if it works better.
+The GNOME desktop environment is highly customizable.  With a few extensions, the desktop can look and feel like many other operating system's desktops.  Launcher bars can be added, window lists can be used, and the overall look and feel of the desktop can be changed.
 
 ---
 
-## User Themes
+## Included Extensions
+
+Pop!\_OS includes the GNOME `Extensions` app by default, which lists both system-installed, and user-installed extensions.
 
 **Please note**: If you are using a custom GNOME theme in Pop_OS! 22.04, it **must** support GTK 4.
```

Comment:
So this is what I've settled on for "Note" blocks: 

Instead of using the word "Note", we'll use a Unicode symbol to represent the type of information provided in the note.

> ℹ️ This is for informational notes.

> ⚠️ This is for cautionary information. Use this if the user should be careful and take the note into consideration before proceeding.